.appveyor.yml

```diff
@@ -14,8 +14,6 @@ branches:
 
 environment:
   matrix:
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      TOOLSET: msvc-14.0
```

Comment:
I suggest to leave that compiler, and only conditionalize the `tensor` builds if that relies on a more recent MSVC version.

Reply:
leaving this to be solved on the boostorg repo.